### PR TITLE
Fixes for validator deployment docs, adding docs for new validators API

### PR DESCRIPTION
**Pull request by kujenga**
I noticed these when looking for a better way to find validators for an account, and while handling the recent testnet restart. Description of changes is in the commit messages. If this would be better submitted in a different form let me know.

### CODE REVIEW

**Comment by PharkMillups** [comment]
Thanks, Aaron! @madninja Can you give this a look?

File: docs/api/blockchain/accounts.mdx [hunk 1]
```diff
@@ -294,6 +294,184 @@ The hotspot list for an account
...
+## Validators for Account
+
+```
+GET https://api.helium.io/v1/accounts/:address/validators
+```
+
+Fetches validators owned by a given account address. The list of returned
+validators is paged. If a `cursor` field is present more results are available.
+
+Note: The cursor for accounts is valid for a limited time. If you receive a
```
Comment (madninja) [inline docs/api/blockchain/accounts.mdx]: This is not the case for the validators for an account. Remove this paragraph

File: docs/api/blockchain/accounts.mdx [hunk 2]
```diff
@@ -294,6 +294,184 @@ The hotspot list for an account
...
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
+_Path Parameters_
+
+| param              | Type     | Note                                     |
+| ------------------ | -------- | ---------------------------------------- |
+| address (required) | _string_ | Account B58 address to fetch details for |
```
Comment (madninja) [inline docs/api/blockchain/accounts.mdx]:
```suggestion
| address (required) | _string_ | Account B58 address to fetch validators for |
```

File: docs/api/blockchain/accounts.mdx [hunk 3]
```diff
@@ -294,6 +294,184 @@ The hotspot list for an account
...
+
+_Query Parameters_
+
+| param             | Type     | Note                                          |
+| ----------------- | -------- | --------------------------------------------- |
+| cursor (optional) | _string_ | Cursor for the next page of hotspots to fetch |
```
Comment (madninja) [inline docs/api/blockchain/accounts.mdx]:
```suggestion
| cursor (optional) | _string_ | Cursor for the next page of validators to fetch |
```

**Comment by kujenga** [comment]
Thanks, I believe those comments have been addressed now

**Review by PharkMillups — APPROVED** [review]

**Comment by PharkMillups** [comment]
Thanks! Merging with these changes. Ping me on Discord if you want the Docs Contributor role.

**Pull request merged**